template/src/api/index.js

```diff
@@ -8,12 +8,21 @@ const logger = require('./middlewares/logger');
 const errorLogger = require('./middlewares/error-logger');
 const config = require('../lib/config');
 {%- set protocol = asyncapi.server(params.server).protocol() %}
-const {{ protocol | capitalize }}Adapter = require('hermesjs-{{protocol}}');
+const serverConfig = {{ protocol | getConfig }};
+const {{ protocol | getProtocol | capitalize }}Adapter = require('hermesjs-{{ protocol | getProtocol }}');
 {%- for channelName, channel in asyncapi.channels() %}
 const {{ channelName | camelCase }} = require('./routes/{{ channelName | convertToFilename }}.js');
 {%- endfor %}
 
-app.addAdapter({{ protocol | capitalize }}Adapter, config.{% if protocol === 'ws' %}ws{% else %}broker.{{protocol}}{% endif %});
+{%- if params.securityScheme and (asyncapi.server(params.server).protocol() === 'kafka' or asyncapi.server(params.server).protocol() === 'kafka-secure') and asyncapi.components().securityScheme(params.securityScheme).type() === 'X509' %}
+const fs = require('fs')
+
+serverConfig.ssl.ca = fs.readFileSync('ca.pem');
```

Comment:
What if the file does not exist? is somehow erroring?

Reply:
added proper try/catch, thanks!